internal/graph/computed/computecheck.go

```diff
@@ -11,14 +11,22 @@ import (
 	v1 "github.com/authzed/spicedb/pkg/proto/dispatch/v1"
 )
 
+type DebugOption int
+
+const (
+	NoDebugging           DebugOption = 0
+	BasicDebuggingEnabled DebugOption = 1
+	TraceDebuggingEnabled DebugOption = 2
```

Comment:
Can we define what basic vs trace means so that users can figure out what the expectation is?

I almost feel like we want to have a set of multiple knobs like "debug_dispatch" which is additive, instead of having "pre-canned levels". But maybe it's just overkill and those levels work for most folks.

Reply:
I can add doc comments, yeah